addons/portal/static/src/scss/portal.scss

```diff
@@ -147,7 +147,7 @@ li > p {
     @extend %o-double-container-no-padding;
 }
 #wrap {
-    > .container, > .container-fluid {
+    .container, .container-fluid {
```

Comment:
because of the "Content Width" option available on all snippets, you also have to add `o_container_small` here, in case the user set the content width of the snippet to "small"

Reply:
Is it mandatory as `.o_container_small` extends `.container` ?